auth/token_test.go

```diff
@@ -108,11 +107,12 @@ func TestTokenRenewWithBadConnection(t *testing.T) {
 		expectedIdempotencyKey: idempotencyKey,
 	}
 
+	uid := int64(1)
 	cm := memoryConfigManager{
 		c: config.Config{
-			AutoConnectData: config.AutoConnectData{ID: 1},
+			AutoConnectData: config.AutoConnectData{ID: uid},
```

Comment:
Here `ID` was set to `1` and then `TokensData` below had token under the key `0`, now it's both `1`. Was it wrong before? :thinking: 
Similar [here](https://github.com/NordSecurity/nordvpn-linux/pull/974/files/80e40dc610ae3915ba196101995b2904af41b96d#diff-64505921a130d998dc87eefe666aabc3bced755658b9313148319b5e1dd53325R242)